tools/test/test.bzl

```diff
@@ -171,31 +175,25 @@ EOF""".format(test_package = test_package),
             resources = resources,
         )
 
-def _common_package(packages):
+def _common_packages(packages):
```

Comment:
`common_packages` is a bit misleading since the function implementation now changed. Maybe `unique_test_packages` makes sense?

Reply:
Updated it to `unique_test_packages`. This function finds the lowest unique packages that actually contains a Test file.